src/pages/TRanimeizle/main.ts

```diff
@@ -1,12 +1,19 @@
 import { pageInterface } from '../pageInterface';
 
-function GetOverviewAnchor(): HTMLAnchorElement | null {
-  return document.querySelector(`a[href^="/anime"]`);
+function GetOverviewAnchor() {
+  const anchor = document.querySelector(`a[href^="/anime"]`) as HTMLAnchorElement;
+
+  if (!anchor) {
+    throw Error("Can't find overview anchor element");
```

Comment:
If the overview anchor couldn't be found on the page, an error will be thrown instead of returning a null result. So that at least will be easy to pinpoint the error from this component